ape_foundry/provider.py

```diff
@@ -674,6 +674,18 @@ def set_code(self, address: AddressType, code: ContractCode) -> bool:
         self._make_request("anvil_setCode", [address, code])
         return True
 
+    def set_storage(self, address: AddressType, slot: int, value: HexBytes):
+        def to_bytes32(val):
+            val_bytes = to_bytes(val)
+            if len(val_bytes) > 32:
+                raise ValueError(f"Value {val} must be <=32 bytes")
+            return HexBytes(b"\x00" * (32 - len(val_bytes))) + val_bytes
+
+        self._make_request(
```

Comment:
This should really have a test because of the `to_bytes32()` logic

Reply:
Can put it in `utils` (or even `ape.utils`) it was really annoying to get this to work, I don't know why `eth-utils` doesn't provide this function